plugins/sql/plugin.spec.yaml

```diff
@@ -4,7 +4,7 @@ products: [insightconnect]
 name: sql
 title: SQL
 description: The SQL plugin allows you to run SQL queries
-version: 3.0.4
+version: 3.0.5
 supported_versions: ["MySQL 8.0.21", "MSSQL 2019 15.0.2000.5", "PostgresSQL 13.0"]
```

Comment:
Can you please add Azure SQL if we are supporting with this version

Reply:
AzureSQL equals MSSQL